### ISSUE TITLE
FAWE as fabric mod

### ISSUE THREAD

**Issue by Cypphi** [IntellectualSites/FastAsyncWorldEdit#656]
Maybe add FAWE fabric support?

This would be useful for single player and fabric servers, but mostly single player. It would be really nice to be able to use FAWE in singleplayer as a builder.

**Comment by N0tMyFaultOG** [comment]
No, bad stalebot.

**Comment by LoboMetalurgico** [comment]
Up!

**Comment by CaseHardened** [comment]
is FAWE not going to  be compatible with fabric?  I see I am late to the party...

**Comment by CuteistCat** [comment]
if not on fabric, at least have quilt

**Comment by JayRPlayz** [comment]
yeah, we need better worldedit on non paper servers

**Comment by StardewEmporer** [comment]
agreed, also i want to play FAWE locally so I can use mods that do not work on servers :/
